core/src/subgraph/registrar.rs

```diff
@@ -275,102 +277,85 @@ where
     }
 }
 
+#[async_trait]
 impl<L, P, S, CS> SubgraphRegistrarTrait for SubgraphRegistrar<L, P, S, CS>
 where
     L: LinkResolver,
     P: SubgraphAssignmentProviderTrait,
     S: Store + SubgraphDeploymentStore,
     CS: ChainStore,
 {
-    fn create_subgraph(
+    async fn create_subgraph(
```

Comment:
It seems that at the moment this does not need to be async. Let's keep it this way though since eventually store will be async.